test/lib/ufe/testDeleteCmd.py

```diff
@@ -187,117 +185,3 @@ def testDelete(self):
         # After the undo we again added two items back.
         self.assertEqual(ufeObs.nbDeleteNotif(), 4)
         self.assertEqual(ufeObs.nbAddNotif(), 4)
-
-    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2011', 'testDeleteArgs only available in UFE preview 2011 and greater')
```

Comment:
Moved this UFE v2 test into separate file included only when UFE v2 is on.

Reply:
Restored test case back in this file and used new `ufeUtils.ufeFeatureSetVersion()` function for when to skip. Removed the new file testDeleteCmd_v2.py.